e2etest/zt_basic_cli_ps_auth_test.go

```diff
@@ -1,176 +1,176 @@
-			os.Setenv("AZCOPY_AUTO_LOGIN_TYPE", "azcli")
-		},
-	}, testFiles{
-		defaultSize: "1K",
-		shouldTransfer: []interface{}{
-			"wantedfile",
-			folder("sub/subsub"),
-			"sub/subsub/filea",
-			"sub/subsub/filec",
-		},
-	}, EAccountType.Standard(), EAccountType.Standard(), "")
-}
-
-func TestBasic_PSAuth(t *testing.T) {
-	RunScenarios(t, eOperation.Copy(), eTestFromTo.Other(common.EFromTo.BlobBlob()), eValidate.Auto(), oAuthOnly, oAuthOnly, params{ // Pass flag values that the test requires. The params struct is a superset of Copy and Sync params
-		recursive: true,
-	}, &hooks{
-		beforeTestRun: func(h hookHelper) {
-			tenId, appId, clientSecret := GlobalInputManager{}.GetServicePrincipalAuth()
-			cmd := `$secret = ConvertTo-SecureString -String %s -AsPlainText -Force;
-				$cred = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList %s, $secret;
-				Connect-AzAccount -ServicePrincipal -Credential $cred`
-			if tenId != "" {
-				cmd += " -Tenant " + tenId
-			}
-
-			script := fmt.Sprintf(cmd, clientSecret, appId)
-			out, err := exec.Command("pwsh", "-Command", script).Output()
-			if err != nil {
-				e := err.(*exec.ExitError)
-				t.Logf(string(e.Stderr))
-				t.Logf(string(out))
-				t.Logf("Failed to login with Powershell " + err.Error())
-				t.FailNow()
-			}
-			os.Setenv("AZCOPY_AUTO_LOGIN_TYPE", "PSCRED")
-		},
-	}, testFiles{
-		defaultSize: "1K",
-		shouldTransfer: []interface{}{
-			"wantedfile",
-			folder("sub/subsub"),
-			"sub/subsub/filea",
-			"sub/subsub/filec",
-		},
-	}, EAccountType.Standard(), EAccountType.Standard(), "")
-}
-
-func TestBasic_PSAuthCamelCase(t *testing.T) {
-	RunScenarios(t, eOperation.Copy(), eTestFromTo.Other(common.EFromTo.BlobBlob()), eValidate.Auto(), oAuthOnly, oAuthOnly, params{ // Pass flag values that the test requires. The params struct is a superset of Copy and Sync params
-		recursive: true,
-	}, &hooks{
-		beforeTestRun: func(h hookHelper) {
-			tenId, appId, clientSecret := GlobalInputManager{}.GetServicePrincipalAuth()
-			cmd := `$secret = ConvertTo-SecureString -String %s -AsPlainText -Force;
-				$cred = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList %s, $secret;
-				Connect-AzAccount -ServicePrincipal -Credential $cred`
-			if tenId != "" {
-				cmd += " -Tenant " + tenId
-			}
-
-			script := fmt.Sprintf(cmd, clientSecret, appId)
-			out, err := exec.Command("pwsh", "-Command", script).Output()
-			if err != nil {
-				e := err.(*exec.ExitError)
-				t.Logf(string(e.Stderr))
-				t.Logf(string(out))
-				t.Logf("Failed to login with Powershell " + err.Error())
-				t.FailNow()
-			}
-			os.Setenv("AZCOPY_AUTO_LOGIN_TYPE", "PsCred")
-		},
-	}, testFiles{
-		defaultSize: "1K",
-		shouldTransfer: []interface{}{
-			"wantedfile",
-			folder("sub/subsub"),
-			"sub/subsub/filea",
-			"sub/subsub/filec",
-		},
-	}, EAccountType.Standard(), EAccountType.Standard(), "")
-}
+//// Purpose: Tests AZCLI and powershell auth tests
```

Comment:
Why are these tests removed? /gen

Reply:
Accidental, realized later. I was running tests locally, these tests panic if you don't have the right tools.